src/Sulu/Bundle/SnippetBundle/Controller/SnippetController.php

```diff
@@ -299,6 +299,10 @@ public function postTriggerAction($uuid, Request $request)
                         $locale,
                         explode(',', $destLocale)
                     );
+
+                    // publish the snippet, otherwise it's not in the live workspace.
+                    $this->documentManager->publish($snippet->getDocument(), $locale);
```

Comment:
That's not working for me. Is there a `$this->documentManager->flush()` missing?